test/resources/SampleProject/SampleProject.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Does this project end up getting built when you the repo? (It shouldn't)

Reply:
It does not. It only ever gets built once it's in the temporary test folder in %temp% as part of it's test.